networking/change-block-size.md

```diff
@@ -152,7 +210,7 @@ calicoctl delete ippool default-ipv4-ippool
 
 #### Create a new IP pool with the desired block size
 
-In this step, we update the IPPool with the new block size of (/28).
+In this step, we update the IPPool with the new block size of (/28) for IPv4. 
```

Comment:
I don't understand this change either - what was wrong with the previous?

Reply:
Reverted